client/cypress/e2e/allocationItemWindowOpen.cy.ts

```diff
@@ -141,6 +152,60 @@ describe('allocation (allocation window open)', () => {
           .should('have.css', 'background-color')
           .and('be.colored', '#f1faf8');
       });
+
+      it('AllocationItem__InputText has correct suffix', () => {
+        cy.get('[data-test=AllocationItem]')
+          .eq(0)
+          .find('[data-test=AllocationItem__InputText__suffix]')
+          .contains('ETH');
+
+        changeMainValueToFiat();
+
+        cy.get('[data-test=AllocationItem]')
+          .eq(0)
+          .find('[data-test=AllocationItem__InputText__suffix]')
+          .contains('USD');
+      });
+
+      it('User can change allocation item value manually ("Use crypto as main value display": true)', () => {
+        cy.get('[data-test=AllocationItem]')
+          .eq(0)
+          .find('[data-test=AllocationItem__InputText]')
+          .clear()
+          .type('0.005');
+        cy.get('[data-test=AllocationItem]')
+          .eq(1)
+          .find('[data-test=AllocationItem__InputText]')
+          .clear()
+          .type('0.002');
+        cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+          .invoke('text')
+          .should('eq', '0.007 ETH');
+        cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+          .invoke('text')
+          .should('eq', '0.003 ETH');
+      });
+
+      it('User can change allocation item value manually ("Use crypto as main value display": false)', () => {
```

Comment:
I think using variables from the store or key from translations (`cryptoMainValueDisplay`) is more universal and would enforce being in sync across the codebase. Imagine changing this string in `translations.json` -- you probably won't take a look into the CY. WDYT?
```suggestion
      it(`User can change allocation item value manually (${IS_CRYPTO_MAIN_VALUE_DISPLAY}: false)`, () => {
```

Reply:
Good idea.